src/sentry/features/__init__.py

```diff
@@ -110,6 +110,7 @@
 default_manager.add("organizations:performance-ops-breakdown", OrganizationFeature)
 default_manager.add("organizations:performance-tag-explorer", OrganizationFeature, True)
 default_manager.add("organizations:performance-tag-page", OrganizationFeature, True)
+default_manager.add("organizations:performance-events-page", OrganizationFeature, True)
```

Comment:
I think you probably also want to configure it in `sentry.conf.server.SENTRY_FEATURES`